engine/model_helpers_test.go

```diff
@@ -7227,16 +7227,17 @@ func TestApitoAccountProfileCase2(t *testing.T) {
 			ExpiryTime:     time.Date(2014, 7, 15, 14, 25, 0, 0, time.UTC),
 		},
 		Weight: 10.0,
-		Balances: []*utils.Balance{{
-			ID:        "VoiceBalance",
-			FilterIDs: []string{"FLTR_RES_GR2"},
-			Weight:    10,
-			Type:      utils.VOICE,
-			Units:     &utils.Decimal{decimal.New(3600000000000, 0)},
-			Opts: map[string]interface{}{
-				"key1": "val1",
-			},
-		}},
+		Balances: map[string]*utils.Balance{
+			"VoiceBalance": {
+				ID:        "VoiceBalance",
+				FilterIDs: []string{"FLTR_RES_GR2"},
+				Weight:    10,
+				Type:      utils.VOICE,
```

Comment:
They type can only be one of *concrete or *abstract. For voice should be *abstract